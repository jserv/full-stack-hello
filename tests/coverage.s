; FIXME: all instructions should be used for coverage test.
add $40 $2 #1
add #1 $8 #2
print #1
print #2
jmp #6
sub #2 $16 #3		; skipped because previous jump
add #2 $100 #3		; to be executed
print #3
mul $-2 $2 #4           ; multiply: (-2)*(2)
mul $-2 $-40 #5         ; multiply: (-2)*(-40)
print #4
print #5
