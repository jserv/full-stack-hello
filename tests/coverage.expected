42
50
150
-4
80
